mx.truffleruby/mx_truffleruby.py

```diff
@@ -132,9 +136,14 @@ def ruby_testdownstream_aot(args):
 
 def ruby_testdownstream_sulong(args):
     build_truffleruby()
-    sulong = mx.suite('sulong')
-    os.environ['SULONG_HOME'] = sulong.dir
-    mx.run(['ruby', 'tool/jt.rb', 'test', '--sulong', ':capi'])
+    # Ensure Sulong is available
+    mx.suite('sulong')
+
+    jt('test', 'cexts')
+    jt('test', 'specs', '--sulong', ':capi')
+    jt('test', 'specs', '--sulong', '-T-Xpatching=false', ':openssl')
+    jt('test', 'mri', '--openssl', '--sulong')
+    jt('test', 'bundle', '--openssl')
```

Comment:
If this doesn't work this is going to break Sulong with no change from them. Have you checked it works?